src/rmp/test/test_blif_writer.blif.ok

```diff
@@ -0,0 +1,8 @@
+.model tmp_circuit
+.inputs inp1 w0
+.outputs w2
+
+
+.gate AND2_X1 A1=inp1 A2=w0 ZN=w1
+.gate OR2_X1 A1=inp1 A2=w1 ZN=w2
+.end
```

Comment:
see that little circle? it means there is a missing return. that is a bug

Reply:
I'm not sure I follow. Can you please elaborate?

Reply:
Hi Ahmad, please add newline after last line of blif. Newline is missing after ".end"